src/main/java/by/mifort/automation/hr/dev/entity/AttributeTypes.java

```diff
@@ -0,0 +1,105 @@
+package by.mifort.automation.hr.dev.entity;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "attributetypes")
```

Comment:
Why you need name here?

Reply:
i have an image of relationships between tables and their names, i just write name like in schema